src/dune_rules/cram/cram_exec.ml

```diff
@@ -430,7 +429,7 @@ let run ~env ~script lexbuf : string Fiber.t =
       in
       Process.create_metadata ~name ~categories:[ "cram" ] ()
     in
-    Process.run ~display:!Clflags.display ~metadata ~dir:cwd ~env Strict sh
+    Process.run ~display:Quiet ~metadata ~dir:cwd ~env Strict sh
```

Comment:
Calling `sh` whilst running a cram test. This is internal and does not need to be reported to the user.

Reply:
When the cram test is invalid (for example a string or a backtick is not closed), we used to see `sh` fail. Does this change the error in that case?

Reply:
It does not change the error, but if - - display=shirt is passed, it doesn't show failing sh to the user. Only the error.

Reply:
ok, sounds good to me then